src/app/commands/cmd_sprite_properties.cpp

```diff
@@ -40,6 +44,95 @@ namespace app {
 
 using namespace ui;
 
+class TilesetListItem : public ui::ListItem {
+public:
+  TilesetListItem(const doc::Tileset* tileset, doc::tileset_index tsi)
+    : ListItem(app::tileset_label(tileset, tsi)),
+    m_buttons(new HBox()) {
+      m_buttons->setTransparent(true);
+      m_buttons->setExpansive(true);
+      m_buttons->setVisible(false);
+
+      auto filler = new BoxFiller();
+      filler->setTransparent(true);
+      m_buttons->addChild(filler);
+
+      auto duplicateBtn = new Button(Strings::sprite_properties_duplicate_tileset());
+      duplicateBtn->setTransparent(true);
```

Comment:
Add the "miniButton" style to buttons (same as in KeyItem in cmd_keyboard_shortcuts.cpp):
```
duplicateBtn->setStyle(theme->styles.miniButton());
```